tutorial/custom-cache-ttl.md

```diff
@@ -7,12 +7,36 @@ nav_exclude: true
 
 # {{page.title}}
 
-_This tutorial was contributed by [Amogh](https://github.com/ags1773)_
+_This tutorial was contributed by [Amogh](https://github.com/ags1773)_ and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/) 
```

Comment:
can we remove underscore before "and" and put it at the end of the the sentence(to make the entire text italic) 
```suggestion
_This tutorial was contributed by [Amogh](https://github.com/ags1773) and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
```

Reply:
done